static_files/css/style.css

```diff
@@ -7,7 +7,34 @@ html {
 }
 
 @import url('https://fonts.googleapis.com/css2?family=Dongle:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Chenla&family=Dongle:wght@300&display=swap');
```

Comment:
Apparently the "@import" is not needed in this case.
```suggestion
```

Reply:
I didn't realize the import included both and didn't inclue the Nunito font family. Thanks!